examples/Makefile

```
@@ -0,0 +1,24 @@
+STYLES := frappe latte macchiato mocha
```

```suggestion
STYLES := latte frappe macchiato mocha
```

This order affects the catwalk generated preview. Although I didn't see it explicitly defined in the style guide anywhere, it seems to be convention that it goes latte -> frappe -> macchiato -> mocha, even in the catwalk previews

---

Ok makes sense that the ordering matters. I never paid attention to it though. Thanks for bringing this up :+1: